File: sdks/thunkmetrc-go/wrapper/models/plants_create_plants_waste_request_item.go
package models

type PlantsCreatePlantsWasteRequestItem struct {
    LocationName string `json:"LocationName,omitempty"`
    MixedMaterial string `json:"MixedMaterial,omitempty"`
    Note string `json:"Note,omitempty"`
    PlantLabels []interface{} `json:"PlantLabels,omitempty"`
    ReasonName string `json:"ReasonName,omitempty"`
    SublocationName string `json:"SublocationName,omitempty"`
    UnitOfMeasureName string `json:"UnitOfMeasureName,omitempty"`
    WasteDate string `json:"WasteDate,omitempty"`
    WasteMethodName string `json:"WasteMethodName,omitempty"`
    WasteWeight float64 `json:"WasteWeight,omitempty"`
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsCreatePlantsWasteRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsCreatePlantsWasteRequestItem(
    @JsonProperty("LocationName")
    val locationName: String? = null,
    @JsonProperty("MixedMaterial")
    val mixedMaterial: String? = null,
    @JsonProperty("Note")
    val note: String? = null,
    @JsonProperty("PlantLabels")
    val plantLabels: List<Any>? = null,
    @JsonProperty("ReasonName")
    val reasonName: String? = null,
    @JsonProperty("SublocationName")
    val sublocationName: String? = null,
    @JsonProperty("UnitOfMeasureName")
    val unitOfMeasureName: String? = null,
    @JsonProperty("WasteDate")
    val wasteDate: String? = null,
    @JsonProperty("WasteMethodName")
    val wasteMethodName: String? = null,
    @JsonProperty("WasteWeight")
    val wasteWeight: Double? = null
)
